### ISSUE TITLE
Alt text/Tooltip is not provided for the “Github” icon control: A11y_Awesome AZD_Landing page_Top navigation_Non-Text content.

### ISSUE THREAD

**Issue by Harika1702** [Azure/awesome-azd#338]
### Tags
#A11yTCS; #A11ySev3; #A11yMAS; #WCP; #CELA P3; #ChromiumEdge; #FTP; #Win11; #awesome-azd-wcp; #CELA_AwesomeAZD_Web_Mar2024; #MAS1.1.1; #Non-Text-Content; #AILimited; #DesktopWeb;

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:
Application: Awesome AZD
URL: https://azure.github.io/awesome-azd/
Chromium Edge: Version 123.0.2420.20
Windows 11 Enterprise.
Version 23H2 OS build 22631.2050.

### Repro Steps:

1. Hit this URL: https://azure.github.io/awesome-azd/.
2. Tab till “Git hub” control at top navigation.
3. Now verify the issue that Alt text/Tooltip is not provided for the “Github” icon control or not.

### Actual Result:
Alt text/Tooltip is not provided for the “Github” icon control.
### Expected Result:
Alt text/Tooltip should be provided for the “Github” icon control.
### User Impact:
Keyboard users will not be able to know the purpose of the control by seeing the icon.
### Recommendation:
Refer [ Developer Resources (sharepoint.com).](https://microsoft.sharepoint.com/sites/accessibility/SitePages/Developer-Resources.aspx)
### MAS Reference:
[MAS 1.1.1 – Non-text Content](https://aka.ms/MAS1.1.1)
### Attachment:

https://github.com/Azure/awesome-azd/assets/100270074/543fcbd9-002e-4856-a99f-af045f96df53

**Comment by Harika1702** [comment]
As this GitHub is universal icon there is no blocking to the users. So, closing this bug. Thanks!
